package.json

```diff
@@ -47,8 +47,10 @@
         "format:check": "prettier --config prettier.config.js --check \"**/*\"",
         "format:fix": "prettier --config prettier.config.js --write \"**/*\"",
         "lint:check": "eslint **/*.{js,ts,tsx}",
-        "lint:check:all": "yarn lint:check && npx lerna run --npm-client yarn lint:check",
+        "lint:check:all": "yarn lint:check && yarn lint:scss && npx lerna run --npm-client yarn lint:check",
         "lint:fix": "eslint **/*.{js,ts,tsx} --quiet --fix",
+        "lint:scss": "stylelint **/*.scss",
+        "lint:scss:fix": "stylelint **/*.scss --fix",
```

Comment:
I think we should consider adding stylelint to the fastpass scripts above. You could do this explicitly or by amending lint:check since that's already included.

Reply:
Great thinking! Added in https://github.com/microsoft/accessibility-insights-web/pull/5314/commits/f560a2bba29be12e0a9ae6227707d509c4da1e6e